src/core/Orb.ts

```diff
@@ -1,82 +1,106 @@
-import { Game } from '~/scenes/Game'
-import { Board } from './Board'
+import { Game } from '~/scenes/Game';
+import { Board } from './Board';
 
 export interface OrbConfig {
-  id: string
-  position: {
-    x: number
-    y: number
-  }
-  radius: number
-  color: number
-  board: Board
-  currCell?: {
-    row: number
-    col: number
-  }
+	id: string;
```

Comment:
I think we'll need a .prettierrc file to enforce whitespace or else we're gonna have a lot of these in the commit diffs